service/es_access.py

```diff
@@ -4,22 +4,31 @@
 
 ELASTICSEARCH_ENDPOINT = app.config['ELASTIC_SEARCH_ENDPOINT']
 MAX_NUMBER_SEARCH_RESULTS = app.config['MAX_NUMBER_SEARCH_RESULTS']
+SEARCH_RESULTS_PER_PAGE = app.config['SEARCH_RESULTS_PER_PAGE']
+
+
+def _get_start_and_end_indexes(page_number):
```

Comment:
If we are receiving 1-based page number in the request, it would probably be best to translate it to 0-based page index and use that index everywhere. Not very strong about it, but we may avoid some potential confusion this way.